src/pages/studyView/charts/pieChart/PieChart.tsx

```diff
@@ -27,6 +27,21 @@ import { DEFAULT_NA_COLOR } from 'shared/lib/Colors';
 import ifNotDefined from '../../../../shared/lib/ifNotDefined';
 import { getTextColor } from 'pages/groupComparison/OverlapUtils';
 
+export function formatPieChartNumber(n: number) {
```

Comment:
@gblaih there is a library we have called numeral.js which should do this.  lets see if we can leverage that.  i think you'll still need these conditions but at least not the math